File: src/gatling/scala/org/openapitools/client/model/ComponentsSchemasIntervalFormsItem.scala

package org.openapitools.client.model


case class ComponentsSchemasIntervalFormsItem (
    /* Form ID scheduled in the interval */
    _formId: Option[Integer],
    /* Form key scheduled in the interval */
    _formKey: Option[String],
    /* Form name scheduled in the interval */
    _formName: Option[String]
)
object ComponentsSchemasIntervalFormsItem {
    def toStringBody(var_formId: Object, var_formKey: Object, var_formName: Object) =
        s"""
        | {
        | "formId":$var_formId,"formKey":$var_formKey,"formName":$var_formName
        | }
        """.stripMargin
}

File: client/php-dt/src/App/DTO/ComponentsSchemasIntervalFormsItem.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\Annotation as DTA;

class ComponentsSchemasIntervalFormsItem
{
    /**
     * Form ID scheduled in the interval
     * @DTA\Data(field="formId", nullable=true)
     * @DTA\Validator(name="Scalar", options={"type":"int"})
     */
    public ?int $form_id = null;

    /**
     * Form key scheduled in the interval
     * @DTA\Data(field="formKey", nullable=true)
     * @DTA\Validator(name="Scalar", options={"type":"string"})
     */
    public ?string $form_key = null;

    /**
     * Form name scheduled in the interval
     * @DTA\Data(field="formName", nullable=true)
     * @DTA\Validator(name="Scalar", options={"type":"string"})
     */
    public ?string $form_name = null;

}
